### ISSUE TITLE
Radio Button group not working properly inside Scrollview

### ISSUE THREAD

**Issue by Codeadmire** [enisn/Xamarin.Forms.InputKit#43]
When I using Radiobuttongroup working correctly in generally but when I using inside scrollview it is not showing selection always keep unselected state only.Please find my structure using below.

The below strucure working fine
 <StackLayout >
< SelectionView SelectionType = "RadioButton" />
</StackLayout >

Not working inside Scrollview
<ScrollView>
 <StackLayout >
< SelectionView SelectionType = "RadioButton" />
</StackLayout >
</ScrollView>

Issue: Always Kept unselected State.

**Comment by enisn** [comment]
Can you share your code ?
It's working in SrollView in the [sample app](https://github.com/enisn/Xamarin.Forms.InputKit/blob/master/Sample.InputKit/Sample.InputKit/MainPage.xaml#L44)

**Comment by Codeadmire** [comment]
Hi @enisn ,

Sorry for the late reply. I found an issues inside `CarouselView` the  

`<cv:CarouselView I>
    <cv:CarouselView.ItemTemplate>
      <DataTemplate>
<ScrollView>
<StackLayout>
< SelectionView SelectionType = "RadioButton" />
</StackLayout>
</ScrollView>
</DataTemplate>`

The above code RadioButton is showing but when I tried to selected the radio button it still maintain their same state but moved Carouselview view forward or backward and come again the same page that time it is showing the last selection button.
But I removed the  scrollview from DataTemplate it is working fine.
I think scrollview not allow to select the RadioButton. But Multichoice selection working fine.
Once issues with RadioButton only.

Thanks,
Tamilmani
